cypress/integration/misc/discount_schema_to_bpartner_spec.js

```diff
@@ -0,0 +1,26 @@
+import { DiscountSchema } from '../../support/utils/discountschema';
+import { BPartner } from '../../support/utils/bpartner';
+
+describe('Create test: discount schema set to bpartner, https://github.com/metasfresh/metasfresh-e2e/issues/113', function() {
+  it('Create discount schema and set it to bpartner', function() {
+    cy.fixture('discount/discountschema.json')
+      .then(discountschemaJson => {
+        Object.assign(new DiscountSchema(), discountschemaJson).apply();
```

Comment:
i think here it's better to add a timestamp to the schema name. to make sure the schema doesn already exis then you create it